utils/git.js

```diff
@@ -1,75 +1,77 @@
-const { resolve } = require('path');
-const REPO = resolve('.');
+const { resolve } = require('path')
+const REPO = resolve('.')
 
-const { execFile } = require('child_process');
+const { execFile } = require('child_process')
 
 function executeGit(cmd, args) {
-  return new Promise((resolve, reject) => {
-    execFile(cmd, args, { cwd: REPO }, (err, stdout) => {
-      if (err) {
-        reject(err);
-      }
-
-      resolve(stdout.toString());
-    });
-  });
+    return new Promise((resolve, reject) => {
+        execFile(cmd, args, { cwd: REPO }, (err, stdout) => {
+            if (err) {
+                reject(err)
+            }
+            // console.log(stdout.toString())
+            resolve(stdout.toString())
+        })
+    })
 }
 
-function parseHistoryItem(line) {
-  const [hash, author, timestamp, msg] = line.split('\t');
+class myGit {
+    constructor() {
```

Comment:
лучше передавать ссылку на executeGit в конструктор

получится такое 
```suggestion
    constructor(executeGit) {
        this.executeGit = executeGit;
    }
```

называется инъекция зависимостей